File: app/src/main/java/com/neome/api/core/user/sig/SigAppVersion.java
// neome.ai API - do not change
//

package com.neome.api.core.user.sig;

import com.neome.api.nucleus.base.sig.Sig;

@SuppressWarnings("unused")
public class SigAppVersion extends Sig
{
  public long currVersionCode;

  public boolean hasForceUpdate;

  public boolean hasUpdate;

  public long mmkvVersion;

  public long sqlVersion;
}

File: app/src/main/java/com/neome/api/core/user/sig/SigAppVersion.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.core.user.sig

import com.neome.api.nucleus.base.sig.Sig

interface SigAppVersion : Sig
{
  val currVersionCode: Long?
  val hasForceUpdate: Boolean
  val hasUpdate: Boolean
  val mmkvVersion: Long?
  val sqlVersion: Long?
}